docker-sync.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+
+# Documentation:
+# https://docker-sync.readthedocs.io/en/latest/getting-started/configuration.html
+
+options:
+  compose-file-path: ${COMPOSE_FILE}
+  verbose: true
+  cli_mode: 'auto'
+  project_root: 'pwd'
+
+syncs:
+  app-magento2-sync:
+    src: ${PROJECT_LOCATION}
+    host_disk_mount_mode: 'cached'
+
+    sync_userid: 1000
+    sync_excludes: ['.git']
+    sync_excludes_type: 'Name'
+
```

Comment:
You can also add this line : `sync_strategy: ${DOCKER_SYNC_STRATEGY}` if someone want to change the strategy

Reply:
Changing the configuration will imply to configure other parameters. In consequence, I don't think it's a good idea to allow this change through a unique variable. Let's keep it as it for now.